docs/userguide/storagedriver/device-mapper-driver.md

```diff
@@ -206,103 +208,151 @@ mode uses block devices to create the thin pool. The following procedure shows
 you how to configure a Docker host to use the `devicemapper` storage driver in
 a `direct-lvm` configuration.
 
-> **Caution:** If you have already run the Docker daemon on your Docker host
+> **Caution:** If you have already run the Engine daemon on your Docker host
 > and have images you want to keep, `push` them Docker Hub or your private
 > Docker Trusted Registry before attempting this procedure.
 
 The procedure below will create a 90GB data volume and 4GB metadata volume to
 use as backing for the storage pool. It assumes that you have a spare block
-device at `/dev/xvdf` with enough free space to complete the task. The device
+device at `/dev/sdd` with enough free space to complete the task. The device
 identifier and volume sizes may be be different in your environment and you
-should substitute your own values throughout the procedure. The procedure also
-assumes that the Docker daemon is in the `stopped` state.
+should substitute your own values throughout the procedure.
 
-1. Log in to the Docker host you want to configure and stop the Docker daemon.
+The procedure also assumes that the Engine daemon is in the `stopped` state.
+Any existing images or data are lost by this process.
 
-2. If it exists, delete your existing image store by removing the
-`/var/lib/docker` directory.
+1. Log in to the Docker host you want to configure.
+2. If it is running, stop the Engine daemon.
+3. Install the logical volume management version 2.
 
-        $ sudo rm -rf /var/lib/docker
+    ```bash
+    $ yum install lvm2
+    ```
+4. Create a physical volume replacing `/dev/sdd` with your block device.
 
-3. Create an LVM physical volume (PV) on your spare block device using the
-`pvcreate` command.
+    ```bash
+    $ pvcreate /dev/sdd
+  ```
```

Comment:
indentation is "off" here; I'll check `make docs`, because I'm not sure indenting fences actually works